packages/spatial/src/renderer/constants/ObjectLayers.ts

```diff
@@ -22,55 +22,59 @@ Original Code is the Infinite Reality Engine team.
 All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
 Infinite Reality Engine. All Rights Reserved.
 */
+import { Opaque } from '@ir-engine/common/src/interfaces/OpaqueType.ts'
```

Comment:
we actually have a duplicate of the opaque type definition, and we need to use the one from hyperflux

Reply:
also, looks like the import has the file extension which can cause problems